docs/version_3.0_differences.md

```diff
@@ -0,0 +1,64 @@
+
+## Behavior differences in the liberty-maven-plugin
+
+### Removed alias goals and renamed goals
+The following goal names are simplified:
+
+| Previous goal name | New goal name |
+| clean-server | clean |
+| create-server | create |
+| debug-server | debug |
+| dump-server | dump |
+| java-dump-server | java-dump |
+| package-server | package |
+| run-server | run |
+| start-server | start |
+| stop-server | stop |
+| test-start-server | test-start |
+| test-stop-server | test-stop |
+
+### Changes in parameter names and defaults for common server parameters
+In the past, the `configFile`, `bootstrapPropertiesFile`, `jvmOptionsFile`, and `serverEnv` [common server parameters](common-server-parameters.md) had default values. If the file existed in the `configDirectory` though, that file took precedence. Now these common server parameters have no default values. If values are specified for these common server parameters, they will take precedence over files located in the `configDirectory`. The `configDirectory` defaults to `${based}/src/main/liberty/config`. Also, `bootstrapProperties` will take precedence over `bootstrapPropertiesFile`, and `jvmOptions` will take precedence over `jvmOptionsFile`. No merging is done.
+
+| Previous parameter name | Previous default value | New parameter name | New default value |
+| configFile | `${basedir}/src/test/resources/server.xml`| serverXmlFile | None | 
+| bootstrapPropertiesFile | `${basedir}/src/test/resources/bootstrap.properties` | bootstrapPropertiesFile | None | 
+| jvmOptionsFile | `${basedir}/src/test/resources/jvm.options` | jvmOptionsFile | None | 
+| serverEnv | `${basedir}/src/test/resources/server.env` | serverEnvFile | None | 
+
+Order of precedence (from highest to lowest):
+
+1) Inlined configuration - bootstrapProperties, jvmOptions
+2) Specified file - bootstrapPropertiesFile, jvmOptionsFile, serverEnvFile, serverXmlFile
+3) File located in `configDirectory` (which defaults to `${basedir}/src/main/liberty/config`)
+
+### Simplified server installation
+In the past, if no `assemblyArchive` or `assemblyArtifact` was specified, the latest Liberty runtime was installed from DHE. Now by default, the latest Open Liberty runtime is installed from Maven Central. The new `liberty.runtime.version` property can be specified on the command line or as a Maven property to override which version of Liberty is installed. Also, `assemblyArchive` is renamed to `runtimeArchive`, and `assemblyArtifact` is renamed to `runtimeArtifact`. These are [common parameters](common-parameters.md) shared by all goals.
+
+### Changed package goal
```

Comment:
The package goal was also updated to remove `packageFile` in favor of using `packageName`, `packageDirectory`, and `packageType`.